### ISSUE TITLE
About training resources and training time

### ISSUE THREAD

**Issue by shuguang-52** [junkunyuan/HAP#4]
Thanks to the author's work for his contribution to the community. I have reproduced SOLIDER before and the original author said it would take 8 V100s and a couple of days. What about that HAP? The author only used half of LUPerson. is this work more friendly for researchers with limited resources (such as only two 80G A100s)?

**Comment by junkunyuan** [comment]
> Thanks to the author's work for his contribution to the community. I have reproduced SOLIDER before and the original author said it would take 8 V100s and a couple of days. What about that HAP? The author only used half of LUPerson. is this work more friendly for researchers with limited resources (such as only two 80G A100s)?

I'm afraid our current work is not very compute efficient, especially when only two A100 GPUs are available. 
To successfully run the pre-training, it would require about 8 A100 (80G) GPUs. This is due to the combination of a large batch size, similar to MAE, and the contrastive loss used in our approach. The estimated time for pre-training would be approximately four days.
